faunadb/client.py

```diff
@@ -11,6 +11,7 @@
 from faunadb.query import _wrap
 from faunadb.request_result import RequestResult
 from faunadb._json import parse_json_or_none, to_json
+from faunadb.streams import Subscription
 
 API_VERSION = "3"
```

Comment:
@trevorsibanda Do we need to bump this to `4` to utilize the new streaming functionality?

Reply:
Bumped api_version to 4. Good catch :+1:

Reply:
Thanks, @trevorsibanda 🙌